nodejs/eks/cluster.ts

```diff
@@ -620,6 +622,28 @@ export function createCore(
         },
     );
 
+    // apply the tags to the cluster security group if it was created by EKS
+    if (!eksClusterSecurityGroup) {
+        const eksCreatedSgId = eksCluster.vpcConfig.clusterSecurityGroupId;
+        eksClusterSecurityGroup = aws.ec2.SecurityGroup.get(
```

Comment:
Does this .get work OK for you inlight of today's discussion about a possible bug in MLC R.Get ?

Reply:
Yeah, this was before our call. I just pushed a new version